fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/CompoundPredicate.java

```diff
@@ -58,7 +58,7 @@ public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
 
     @Override
     public Expression withChildren(List<Expression> children) {
-        return new CompoundPredicate(getType(), children.get(0), children.get(1));
+        throw  new RuntimeException("Do not implement this method");
```

Comment:
Use `Do not implement this method` as a comment. For the exception message, I think it is better to use another description.